### ISSUE TITLE
[ISSUE]Cannot Print to Local Network Printer While Connected to ProtonVPN

### ISSUE THREAD

**Issue by mitcon3** [ProtonVPN/linux-cli#211]
**Describe the bug**
Cannot print to local network printer while pvpn connection is active. Print job remains in queue (Hold status). After disconnecting from pvpn print print jobs can be released and printing continues normally. Have tried Split Tunneling with specific address of printer (192.168.1.22) and a local network range (192.168.1.0/24), jobs still in hold status when printing.

**To Reproduce**
Connect via pvpn c -f command
Attempt to print a local text file
Print Job remains in a Hold status

**Expected behavior**
Ability to print locally while connected to pvpn.

**Error Messages/Program Output/Log Messages (~/.pvpn-cli/pvpn-cli.log)**
No error messages. If you can point me to specific text patterns to search for in pvpn-cli.log, I would be happy to do that.

**Desktop (please complete the following information):**
 - OS: Debian 9.12
 - Python Version: 3.5.3
 - ProtonVPN-CLI Version: 2.2.4

**Additional context**
Have attached output from "ip route" command, connect.ovpn and split_tunnel.txt. I have questions about the route line in connect.ovpn, specifically why the mask is set to 255.255.255.255 instead of 255.255.255.0; and how can I see what the value of the net_gateway is.

[route_table.txt](https://github.com/ProtonVPN/linux-cli/files/4928286/route_table.txt)

**Comment by Rafficer** [comment]
Is the Kill Switch enabled?

**Comment by mitcon3** [comment]
Rafficer:

No, only Split Tunneling.
BTW, nice cli! Love the simplicity!

Mitch

**Comment by Rafficer** [comment]
Can you `ping 192.168.1.22` while being connected? It may be just that your OS tries to access the printer via it's name and since DNS points to ProtonVPN's servers, it can't resolve your local printer.

**Comment by mitcon3** [comment]
Rafficer:

Ping works...

I hear what you're saying, makes sense!

I think I should figure out how Debian addresses the printers and how to get it to address them via IP address. This would explain why my NAS drives are working. They are mounted via IP address ... no "symbolic" names. Will try to get this accomplished this weekend.

Thanks for the idea!
Mitch

**Comment by mitcon3** [comment]
Rafficer:

I should have started chasing this problem down over here. Spent a couple weeks working with someone on the ProtonVPN blog, only to be referred here.

You were exactly right! When I looked at the printer setup in the CUPS Utility I found all sorts of references to DNS in the connection setting. I setup a new printer and got the connection setting to be "socket://192.168.1.22:9100...BANG it worked the first time!

AWESOME! Thanks for pointing me in the right direction!
Mitch
